exasol_sphinx_github_pages_generator/generate_index.py

```diff
@@ -108,7 +108,31 @@ def get_releases(target_branch: str, target_branch_exists_remote: bool, source_b
     return release_list_dicts
 
 
-def generate_release_index(target_branch: str, target_worktree: Path, source_branch: str, target_branch_exists_remote: bool) -> None:
+def copy_importlib_resources_file(src_file, target_file):
+    try:
+        content = src_file.read_text()
+        with open(target_file, "w") as file:
+            file.write(content)
+    except UnicodeDecodeError:
+        content = src_file.read_bytes()
```

Comment:
I think, you can use read_bytes always, because you actually don't need the content encoded

Reply:
changed